install/kubernetes/cilium/templates/cilium-agent/daemonset.yaml

```diff
@@ -43,11 +43,6 @@ spec:
         # ensure pods roll when configmap updates
         cilium.io/cilium-configmap-checksum: {{ include (print $.Template.BasePath "/cilium-configmap.yaml") . | sha256sum | quote }}
         {{- end }}
-        # This annotation plus the CriticalAddonsOnly toleration makes
-        # cilium to be a critical pod in the cluster, which ensures cilium
-        # gets priority scheduling.
-        # https://kubernetes.io/docs/tasks/administer-cluster/guaranteed-scheduling-critical-addon-pods/
-        scheduler.alpha.kubernetes.io/critical-pod: ""
```

Comment:
Consider that cilium requires minimum k8s version as 1.16+ (except cilium 1.19 as per [docs](https://docs.cilium.io/en/v1.9/concepts/kubernetes/requirements/)), it's ok to clean up this annotation in my opinion.

If it's not much of issue, can you help to remove this annotation in test/k8sT/manifests/log-gatherer.yaml as well?

> the scheduler.alpha.kubernetes.io/critical-pod annotation is removed. Pod priority (spec.priorityClassName) should be used instead to mark pods as critical. (#80342, @draveness)

https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.16.md

Reply:
@sayboras sounds fine. I am surprised there is such a liberal versioning policy. Supporting many Kubernetes versions is very difficult in a fast moving project. I am interested in the release schedule. Can you point me at the governance repo for this project? I didn't see it in cilium/community or cilium/governance.

Cheers,
-steve

Reply:
It's a great question, I think that it's always a fine balance between promoting young project (for better adoption) vs maintenance effort of supporting old k8s versions.

I have very limited knowledge on this matter, so probably need to wait for other team members to chime in.

Reply:
cc @cilium/cilium-maintainers

Reply:
Hi @sdake , the governance of the project is documented here: https://docs.cilium.io/en/stable/contributing/governance/ , and we mostly conduct the community interactions either here on GitHub or over on Slack (link on the main website). 

Regarding k8s versions, we don't really anticipate that users _should_ be running such old versions, but let's say that we release v1.9 to support the 3 k8s versions which were supported upstream at that time (nov 2020), that meant v1.17-v1.19. We won't update the testing infrastructure and so on for these older branches, so they will continue to test against the supported versions of the time. Furthermore, users need to upgrade both their k8s versions and their Cilium versions, so unless they are strict about upgrading these in lockstep on a very regular basis, it could make upgrades more difficult for users if there were a more narrow set of versions that Cilium _can_ work against. Beyond this, some clouds have been a bit slow to update, and also for the most part, there hasn't been a large amount of work on the Cilium project side to support the older versions. So we haven't bumped the minimum versions too frequently. I think that on average we could expect to roughly release Cilium once per k8s release cycle, and drop the oldest supported version in each release while allowing users to upgrade their Cilium versions on their older clusters and then upgrade their k8s versions.

If you have any other particular information you would like to know about the governance of the project, feel free to file an issue as a question or ask on Slack and we can see about getting the information you are interested in, and potentially improving on the documentation of the governance processes.